packages/ansifmt/ansifmt.0.1.3/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "A simple, lightweight library for ANSI formatting"
+description:
+  "A simple, lightweight library for ANSI formatting with powerful features such as a tokenization-based system for pretty-printing code in the terminal."
+maintainer: ["Qexat <contact@qexat.com>"]
+authors: ["Qexat <contact@qexat.com>"]
+license: "MIT"
+tags: ["ansi" "formatting" "pretty-printing" "terminal"]
+homepage: "https://github.com/qexat/ansifmt"
+bug-reports: "https://github.com/qexat/ansifmt/issues"
+depends: [
+  "dune" {>= "3.17"}
+  "ocaml"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
```

Comment:
The build is failing on older compilers. Are you OK with adding

```suggestion
    "dune" { >= "4.08" }
```

and publishing this as is, or would you rather make updates for compat with the older compilers?

I recommend the former, since the repo will soon not support packages that can't build with at least 4.08.

Reply:
Should this be `"ocaml" { >= "4.08" }`?

Reply:
> The build is failing on older compilers. Are you OK with adding
> 
> and publishing this as is, or would you rather make updates for compat with the older compilers?
> 
> I recommend the former, since the repo will soon not support packages that can't build with at least 4.08.

I don't really care about supporting OCaml <5 so that's fine to me

Reply:
Oops, yes, thanks, @yawaramin !